src/ValueParsers/IsoTimestampParser.php

```diff
@@ -48,7 +49,7 @@ public function __construct(
 	) {
 		parent::__construct( $options );
 
-		$this->defaultOption( self::OPT_CALENDAR, self::CALENDAR_GREGORIAN );
+		$this->defaultOption( self::OPT_CALENDAR, null );
```

Comment:
Note: We can set this option to `CALENDAR_GREGORIAN` in Wikibase.git, if we need to avoid a breaking change.